### PR TITLE
Adds multiple hosts documentation

### PR DESCRIPTION
**Pull request by paganotoni**
This PR adds an example of specifying multiple hosts within the proxy host field.

ref: https://github.com/basecamp/kamal-proxy/issues/14

### CODE REVIEW

**Comment by marckohlbrugge** [comment]
Just wanted to say thank you 🙏 – Just ran into this question and you solved it 7 minutes ago

File: docs/configuration/proxy.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ If no hosts are set, then all requests will be forwarded, except for matching re
 ```yaml
   host: foo.example.com
 ```
+If multiple hosts are needed, these can be specified by comma separating the hosts.
+
+```yaml
+  host: foo.example.com,bar.example.com
```
Comment (nickhammond) [inline docs/configuration/proxy.md:33]: Ooo this is cool, nice!

**Comment by djmb** [comment]
Oh! That will work won't it! I'm planning to add support for a hosts field so you can specify them as:

```
proxy:
  hosts:
    - a.example.com
    - b.example.com
```

**Comment by paganotoni** [comment]
Yes, @djmb, it works! However, I like the alternative you posted; it's cleaner. Are wildcard hosts supported?

**Comment by lucasmeijer** [comment]
> Yes, @djmb, it works! However, I like the alternative you posted; it's cleaner. Are wildcard hosts supported?

Having wildcard hosts would be really amazing.    I have like 100 subdomains.

**Pull request merged**